terraform/modules/lambda/files/cronMailing/cronMailing.py

```diff
@@ -0,0 +1,12 @@
+import json
+import os
+import requests
+
+def lambda_handler(event, context):
+    print("*** START MAILING CRON JOB ***")
+    
+    res = requests.post(f'{os.environ.get("API_ENDPOINT")}/mail', data = { "key": os.environ.get("LAMBDA_SECRET_KEY") })
+    print(res.text)
+
+    print("*** END MAILING CRON JOB ***")
+    
```

Comment:
add newline for eof plz

Reply:
isn't there a newline already?

Reply:
I thought so as well 👀 
![image](https://user-images.githubusercontent.com/44826218/143661417-80efe77a-073c-4281-ba49-554fba1bed4a.png)